index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <style>
+        @import url('https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap');
+    </style>
+    <title>Prototype Mobile Portafolio</title>
+</head>
+<body>
+    <main class="container">
+        <header>
+            <h3>WELCOME</h3>
+            <section class="nav-bar">
+                <div class="line"><h3></h3></div>
+                <div class="line"><h3></h3></div>
+                <div class="line"><h3></h3></div>
+            </section>
+        </header>
+        <section class="presentation">
+            <h2 class="introduction">Hey there. I’m Rafael</h2>
+            <h2 class="introduction-second-part">I’m a software developer</h2>
+            <p class="supporting-text"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hestiate to contact me.</p>
+            <div class="icon-social">
+                <img src="images/twitter.png" class="social" alt="twitter logo">
+                <img src="images/Linkedin.png" class="social" alt="Linkedin logo">
+                <img src="images/medium.png" class="social" alt="Medium logo">
+                <img src="images/github.png" class="social" alt="Github logo">
+                <img src="images/angellist.png" class="social" alt="angellist logo">
+            </div>
```

Comment:
Please change the `div` tag to be `ul` tag and wrap each `img` tag by `li` tag for semantic purposes.